packages/perseus/src/widgets/categorizer/categorizer-validator.ts

```diff
@@ -0,0 +1,34 @@
+import type {Rubric, UserInput} from "./categorizer.types";
+import type {PerseusStrings} from "../../strings";
+import type {PerseusScore} from "@khanacademy/perseus";
```

Comment:
We should avoid importing from `@khanacademy/perseus` when we're within `@khanacademy/perseus`. The solution is relative imports.

Reply:
🤦‍♀️ Thank you! Fixed!